core/src/main/java/io/kestra/core/runners/VariableRenderer.java

```diff
@@ -128,14 +128,17 @@ public String recursiveRender(String inline, Map<String, Object> variables) thro
             isSame = currentTemplate.equals(current);
             currentTemplate = current;
         }
+        while(!isSame && !isVerbatim(inline));
 
         return current;
     }
 
-    public IllegalVariableEvaluationException properPebbleException(PebbleException e) {
-        if (e instanceof AttributeNotFoundException) {
-            AttributeNotFoundException current = (AttributeNotFoundException) e;
+    private boolean isVerbatim(String inline) {
```

Comment:
will only work with a raw starting with `{%verbatim` and will failed with `content{%verbatim` that is a must have.